core/point/point.go

```diff
@@ -74,6 +74,11 @@ func (pt *Point) ToSGF() (string, error) {
 	return sgfOut, err01
 }
 
+// String() method to represent and print a Point, useful for debugging and test purposes
+func (pt Point) String() string {
+	return fmt.Sprintf("Point(%d,%d)", pt.x, pt.y)
```

Comment:
optional: in go, typically you would just put the data `{x,y}` rather than the type. I think that's a little more compact.